lib/greet.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.sayHi = function(name){
+  return 'Hello ' + name + '!';
+};
+
+exports.askName = function(){
+   const readline = require('readline');
+   const rl = readline.createInterface(process.stdin, process.stdout);
+
+   rl.question('What is your name? ', (answer) => {
+    console.log(`Hello ${answer} Nice to meet you!!`);
+    rl.close();
+   });
+ };
```

Comment:
Good job in using the docs to find a creative way to solve the the bonus using readline and arrow functions! I like how you were able to create an interactive take on the bonus. Another way this could have been handled was through the use of process.argv[2]. argv is an array of arguments on the global process object, in reference to node, the "first" command line argument starts at the second index on the argv array. The 0th index is always node and the 1st index is always the <filename> In this way, you can use process.argv[2] to also dynamically generate a name for your greet method.